src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -436,9 +452,45 @@ def mousePressEvent(self, event):
             This method is called to handle a mouse press event
             :param event: the mouse event
         """
+        print("mousePress even called with event: " + str(event))
+        if self.cursor:
+            self.removeItem(self.cursor)
+        self.isPressed = True
+        if (
+                2 * QtGui.QVector2D(event.pos() - self.rect.center()).length()
+                < self.rect.width()
+        ):
+            self.drag_position = event.pos() - self.rect.topLeft()
+        super().mousePressEvent(event)
+        x, y = linear_transform(
+            math.floor(event.scenePos().x()), math.floor(event.scenePos().y()),
+            self.rows, self.cols)
+        is_coloured = (x, y) in self.according_color_dict
+        self.is_current_pixel_coloured = is_coloured
+        self.draw_cursor(event.scenePos().x(), event.scenePos().y(),
+                         self.draw_tool_radius, new_circle=True)
```

Comment:
This block is hard to read, consider better variable names for readability. Or spacing, simple comments etc.

Reply:
I have no clue about this as I didn't write it, have made it as readable as I can by fixing the spacing

Reply:
aight fair